src/api/application/application.ts

```diff
@@ -140,13 +140,20 @@ export default class ApplicationModule extends Base {
         await this.wire.sendAction('create-application', appOptions);
         return await this.wrap({ uuid: appOptions.uuid });
     }
+
+    /**
+    * DEPRECATED method to creat a new Application. Use {@link Application.start} instead.
+    * @param { ApplicationOption } appOptions
```

Comment:
"create" - also - do we want to document this? @rdepena @pbaize ?

Reply:
Good catch. +1 on the question. My thinking was that it would be good to have since the method does exist, may be used in legacy code / examples, etc. But also makes sense to remove it if that's the preference.